### DISCUSSION TITLE
pause plotting

### DISCUSSION THREAD

**Discussion by rstyle0354** [Chia-Network/chia-blockchain#6451] [Support]
Good afternoon everyone, friends.

On the fourth day, plotting stops at the 3rd stage of compressing. I paddle on one raft, there are no parallel ones. First I used a standard chia program, then switched to PowerShell. I tested it ". \ Chia plots create -k 32 -b 3000 -r 4 -t F: \ 4 \ -d D: \ full" so temporary files are stored in sata SSD 2Tb actually shows 1.81 Tb then I thought that enough space and added an add-on like this ". \ chia plots create -k 32 -b 3000 -r 4 -t F: \ 4 \ -2 E: \ -d D: \ full" where E: I have nvme 256 GB again the same fi .... Then I switched to Harry Plotter, it also stops there, even waited a day, they say, start, no. The system works stably and does not freeze.

Here's the PC setup:

Intel Core i5 9400F 6 cores 6 threads

RAM 8 GB

Sata SSD 128GB system

Sata SSD 2 TB for cache

HDD 3 TB for ready-made rafts

NVME 256GB optional

Mother MSI B365M PRO-VDH

OS- win10

I can't understand what's the matter

**Comment by BrandtH22** [comment]
@rstyle0354 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
